goblint.opam.locked

```diff
@@ -64,10 +64,10 @@ depends: [
   "mlgmpidl" {= "1.2.14"}
   "num" {= "1.4"}
   "ocaml" {= "4.14.0"}
-  "ocaml-base-compiler" {= "4.14.0"}
+  "ocaml-variants" {= "4.14.0+options"}
   "ocaml-compiler-libs" {= "v0.12.4"}
   "ocaml-config" {= "2"}
-  "ocaml-options-vanilla" {= "1"}
+  "ocaml-option-flambda" {= "1"}
```

Comment:
Should we possibly consider creating a separate file, specifically for flambda (i.e., `make setup` would use one configuration of the project while `make setup-flambda` would use a different one)?

Reply:
There can be just one opam lock file for each opam package, so that won't work.
Since the performance difference is significant, I think we just need to decide ourselves what should be the default such there's consistency across the board when it comes to benchmarking.

Reply:
Can we consider this resolved?